aiohttp/web.py

```diff
@@ -133,6 +135,8 @@ def _set_loop(self, loop):
         if self._debug is ...:
             self._debug = loop.get_debug()
 
+        self._jobs = JobRunner(loop=loop)
```

Comment:
Naming bikeshedding alert. 

Jobs is a JobRunner, not a list of jobs which you might think following you intuition. I always mess with `app.on_startup` / `app.on_cleanup` callbacks because these are not a callable accumulators, but list-like objects.

May be use `scheduler` word and have Scheduler class instead of JobRunner? Yeap, a bit longer, but quite clear what it about. (Personally, I just don't like Job and any Runners).